src/partial_sheet_signatures.js

```diff
@@ -0,0 +1,128 @@
+import {Booklet} from './booklet.js';
+
+export class PartialSheetSignatures {
```

Comment:
I did a test export of this and the zip downloaded was empty-- does this work for you locally?